Makefile

```diff
@@ -1,16 +1,12 @@
 PACKAGES = notesserver notesapi
 .PHONY: requirements
 
-validate: test.requirements test coverage
+include .travis/docker.mk
 
-ifeq ($(ESVER),-)
-test_settings = notesserver.settings.test_es_disabled
-else
-test_settings = notesserver.settings.test
-endif
```

Comment:
@fredsmith I think different setting could be the reason for failure, not sure why we removed these settings.